### ISSUE TITLE
Replace vertical lines that accompany cursor with a single vertical line 

### ISSUE THREAD

**Issue by curran** [sul-cidr/josquin-ribbon#14]

**Comment by seemantk** [comment]
In this implementation, the brush widget (to zoom and pan along the minimap) is different from the visualization on which this ticket was based.  In that visualization, two vertical lines appeared whenever the mouse was hovered over the notes.  Those two lines seem to define an interval in which to see.

In this implementation, nothing appears on hover (except for tooltips), and the brush needs to be dragged to create the zoom effect.

In other words, the mechanics of our implementation are different enough from the previous one that this issue isn't relevant.

@craigsapp and @gworthey: do you agree?

**Comment by curran** [comment]
I believe we discussed this during one of our calls, and the Josquin team agreed that the line is no longer necessary. Am I remembering correctly?

@seemantk What do you think, should we add a vertical line that is linked between the focus and context views, or should we go ahead and close this issue without making any changes?

**Comment by seemantk** [comment]
Agreed, let's close this issue.
